### PR TITLE
Enforce no TODOs

### LINKED ISSUE #2885: Do not apply TODO docs replaces when the TODO is not fixed
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

We apply replaces for lines that contain "terraform", such as:

```json
  "website/docs/d/default_tags.html.markdown": [
    {
      "old": "With this data source, you can apply default tags to resources not _directly_ managed by a Terraform resource, such as the instances underneath an Auto Scaling group or the volumes created for an EC2 instance.",
      "new": "With this data source, you can apply default tags to resources not _directly_ managed by a TODO resource, such as the instances underneath an Auto Scaling group or the volumes created for an EC2 instance."
    }
  ]
```

We shouldn't apply any changes unless the `"new"` has the `"TODO"` replaced with a new value. That way replaces that have not been addressed will be omitted from our docs, instead of filled with confusing values.

### PR DESCRIPTION
**Pull request by iwahbe**
Fixes #2885

I removed all unused replacements from `replacements.json`, added a new replace rule to reduce the number of TODOs that needed to be manually fixed up. I then fixed up all remaining TODOs. https://github.com/pulumi/pulumi-aws/pull/2892/commits/324d355935664f036b95a7eac939a22ee8d7b5af prevents unaddressed TODOs from showing up in our docs.

### CODE REVIEW

File: sdk/dotnet/CleanRooms/ConfiguredTable.cs [hunk 1]
```diff
@@ -32,15 +32,15 @@ namespace Pulumi.Aws.CleanRooms
...
+    ///         Description = "I made this table with Pulumi!",
     ///         TableReference = new Aws.CleanRooms.Inputs.ConfiguredTableTableReferenceArgs
     ///         {
     ///             DatabaseName = "example_database",
     ///             TableName = "example_table",
     ///         },
     ///         Tags = 
     ///         {
-    ///             { "Project", "TODO" },
+    ///             { "Project", "Pulumi" },
```
Comment (t0yv0) [inline sdk/dotnet/CleanRooms/ConfiguredTable.cs:43]: Nice

File: sdk/java/src/main/java/com/pulumi/aws/lightsail/inputs/BucketState.java [hunk 2]
```diff
@@ -78,14 +78,14 @@ public Optional<Output<String>> createdAt() {
     }
 
     /**
-     * Force Delete non-empty buckets using `TODO destroy`. AWS by default will not delete an s3 bucket which is not empty, to prevent losing bucket data and affecting other resources in lightsail. If `force_delete` is set to `true` the bucket will be deleted even when not empty.
+     * Force Delete non-empty buckets using `pulumi down`. AWS by default will not delete an s3 bucket which is not empty, to prevent losing bucket data and affecting other resources in lightsail. If `force_delete` is set to `true` the bucket will be deleted even when not empty.
```
Comment (t0yv0) [inline sdk/java/src/main/java/com/pulumi/aws/lightsail/inputs/BucketState.java:81]: TIL pulumi down.

**Review by t0yv0 — APPROVED** [review]
Looks useful. Spot-checking example changes looks good to me.

File: sdk/java/src/main/java/com/pulumi/aws/lightsail/inputs/BucketState.java (hunk 2 above)
Comment (iwahbe) [reply sdk/java/src/main/java/com/pulumi/aws/lightsail/inputs/BucketState.java:81]: That was a mistake. It should be `pulumi destroy`, but `pulumi down` does work.

**Pull request merged**